### PR TITLE
🚀Remove GOV.UK template

### PR DESCRIPTION
**Pull request by katstevens**
https://trello.com/c/5RIlSNJ9/261-2-remove-govuktemplate-from-brief-responses-frontend

Based on https://github.com/alphagov/digitalmarketplace-supplier-frontend/pull/1159.

~Caveat: I have not QAed this in the slightest yet but wanted to put it up for early review (hence Draft).~ Now tested!

### CODE REVIEW

**Comment by katstevens** [comment]
Spotted a couple of errors - will push fixes shortly

File: app/templates/briefs/start_brief_response.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "text": brief.title
       },
       {
-        "text": "Apply for ‘{}’".format(brief.title)
+        "text": "Apply for this opportunity"
```
Comment (lfdebrux) [inline app/templates/briefs/start_brief_response.html]: 💯

**Comment by katstevens** [comment]
Tests now fixed and manual QA done - this is ready, I think 😸

**Comment by lfdebrux** [comment]
Yep, looks good for me locally, let's merge it

**Pull request merged**